Source/PeleLMeX_Advance.cpp

```diff
@@ -253,6 +258,47 @@ PeleLM::Advance(int is_initIter)
   }
 }
 
+void
+PeleLM::WriteOutflowPlane(int a_step, Real a_time) const
+{
+  if (
+    m_write_outflow_plane >= 0 && m_write_outflow_plane < 2 * AMREX_SPACEDIM &&
+    a_step >= 0 && a_step % m_write_outflow_plane_int == 0) {
+    const int outflow_dir = m_write_outflow_plane % AMREX_SPACEDIM;
+    const Orientation::Side outflow_face =
+      m_write_outflow_plane >= AMREX_SPACEDIM ? Orientation::high
+                                              : Orientation::low;
+    const Orientation orient(outflow_dir, outflow_face);
+
+    Box outflow_box = amrex::adjCell(geom[0].Domain(), orient, 1);
```

Comment:
How about generalizing this to dump from intermediate planes as well? If we dump data directly on boundaries the data may be affected somewhat by outflow BCs (if non-periodic).

Reply:
Sure, but then we'd have to rename everything! :). My vote is to leave as is and generalize when the need arises, only because it's slightly harder to make sure it's right, and complicates the specification a bit.

Unrelated followup though...in my tests, I chopped data from EB_PipeFlow, but shoved it into a non-EB case.  It "worked" but gave some very odd looking results that might have just reflected the lack of care in ensuring a physical union of the two runs (properly dealing with inflow+transport, etc).

Reply:
or.... could do `plane_dir`, `plane_loc`.  IF that works, I could fix things.

Reply:
Yeah that's what I was thinking. Just have `plane_loc` be an int that indicates the coarse grid index of the plane being dumped. Or a real and then compute the int index.

Reply:
Ok, should be ready